ext_localconf.php

```diff
@@ -1,7 +1,5 @@
 <?php
 
-use Mindscreen\JsonReports\Command\ReportsCommandController;
-use Mindscreen\JsonReports\Controller\ReportsController;
```

Comment:
Keine "use"  statments in ext_localconf.php (erst ab TYPO3 11.4)

da TYPO3 die dateien einfach "concatiniert" scheitert es falls zwei ext_localconf.php files die gleichen Classen"usen"

https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/ConfigurationFiles/Index.html

Reply:
Ist angepasst (und überflüssige Config für HTTP Abfrage entfernt)